.github/workflows/dockerimage.yml

```diff
@@ -27,9 +30,14 @@ jobs:
       id: package-version
       uses: martinbeentjes/npm-get-version-action@v1.0.0
 
+    - uses: actions/checkout@v2
+      with:
+        ref: v${{ steps.package-version.outputs.current-version }}
```

Comment:
Since there are two repos here, it's better to speficy their paths separately to avoid confusion, and specify working directory in subsequent steps:

```yml
- name: Checkout master repo
    uses: actions/checkout@v2
    with:
      repository: $REPO
      ref: ...
      path: ./master

- name: Checkout release repo
    uses: actions/checkout@v2
    with:
      repository: $REPO
      ref: ...
      path: ./release

- name: Subsequent steps
    working-directory: ./master OR ./release
    run: .....
```